creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -778,6 +782,7 @@ class SnapbackSM {
           for (const secondary of secondaries) {
             const secUserSyncSuccessRate = userSecondarySyncMetrics[secondary]['SuccessRate']
             if (secUserSyncSuccessRate < this.MinimumSecondaryUserSyncSuccessPercent || unhealthyPeers.has(secondary)) {
+              this.log(`processStateMachineOperation(): Secondary ${secondary} for user ${nodeUser.wallet} has userSyncSuccessRate of ${secUserSyncSuccessRate}; failed to meet threshold of ${this.MinimumSecondaryUserSyncSuccessPercent} - found value ${secUserSyncSuccessRate}. Marking replica as unhealthy.`)
```

Comment:
this is purely so we have more visibility into the scenario where a secondary has failed to meet minimum sync success % as this should be rare